README.md

```diff
@@ -9,16 +9,9 @@ Kani verifies:
  * The absence of panics (e.g., `unwrap()` on `None` values)
  * The absence of some types of unexpected behavior (e.g., arithmetic overflows)
 
-## GitHub Action
-
-Use Kani in your CI with `model-checking/kani@VERSION`. See the
-[GitHub Action section in the Kani
-book](https://model-checking.github.io/kani/install-github-ci.html)
-for details.
-
 ## Installation
 
-To install the latest version of Kani, run:
+To install the latest version of Kani ([Rust 1.58+; Linux or Mac](https://model-checking.github.io/kani/install-guide.html)), run:
```

Comment:
Is it all linux? Or just tested on ubuntu?

Reply:
I don't think we've heard of any compatibility problems from others. We have tested on AL2, NixOS at least. I believe I also tested on CentOS 7.

But I'd definitely also state anything more specific on the install page, I think the README needs to stay really concise.